tests/phpunit/api/v3/ActivityTest.php

```diff
@@ -1403,4 +1403,34 @@ public function testGetWithOr() {
     $this->assertEquals(3, $result['count']);
   }
 
+  public function testGetOverdue() {
+    $overdueAct = $this->callAPISuccess('Activity', 'create', array('activity_date_time' => 'now - 1 week', 'status_id' => 'Scheduled') + $this->_params);
+    $completedAct = $this->callAPISuccess('Activity', 'create', array('activity_date_time' => 'now - 1 week', 'status_id' => 'Completed') + $this->_params);
```

Comment:
These two lines are quite long and hard to read. What about formatting them in the same style as the `get` calls a few lines bellow?